sysl2/sysl/visitor.go

```diff
@@ -207,17 +226,21 @@ func (v *SequenceDiagramVisitor) Visit(e Element) error {
 			log.Errorln(err)
 		}
 	}()
-
+	var err error
```

Comment:
Not required. Just use `return …` in each `case` clause.

Reply:
Please see the comment below.

Reply:
This can't be moved below the for loop, as the bbVal.VisitCount is calculated in the call v.visitEndpointCollection(t).